aws-ssm-patchbaseline/docs/README.md

```diff
@@ -0,0 +1,230 @@
+# AWS::SSM::PatchBaseline
```

Comment:
why is this readme needed?
the information is not informative, are they auto generated?

Reply:
This is deafult startup file and its stands for application usage information and now we reverted the changes in pull request